packages/graphql/src/schema/generation/get-mutation-input-from-attribute-type.ts

```diff
@@ -0,0 +1,182 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { GraphQLInputType } from "graphql";
+import {
+    BigIntListMutations,
+    BigIntScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/BigIntScalarMutations";
+import {
+    BooleanListMutations,
+    BooleanScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/BooleanScalarMutations";
+import {
+    CartesianPointListMutations,
+    CartesianPointMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/CartesianPointMutations";
+import {
+    DateListMutations,
+    DateScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/DateScalarMutations";
+import {
+    DateTimeListMutations,
+    DateTimeScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/DateTimeScalarMutations";
+import {
+    DurationListMutations,
+    DurationScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/DurationScalarMutations";
+import {
+    FloatListMutations,
+    FloatScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/FloatScalarMutations";
+import {
+    IDListMutations,
+    IDScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/IDScalarMutations";
+import {
+    IntListMutations,
+    IntScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/IntScalarMutations";
+import {
+    LocalDateTimeListMutations,
+    LocalDateTimeScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/LocalDateTimeScalarMutations";
+import {
+    LocalTimeListMutations,
+    LocalTimeScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/LocalTimeScalarMutations";
+import {
+    PointListMutations,
+    PointMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/PointMutations";
+import {
+    StringListMutations,
+    StringScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/StringScalarMutations";
+import {
+    TimeListMutations,
+    TimeScalarMutations,
+} from "../../graphql/input-objects/generic-mutation-operations/TimeScalarMutations";
+import type { AttributeAdapter } from "../../schema-model/attribute/model-adapters/AttributeAdapter";
+
+export function getMutationInputFromAttributeType(attribute: AttributeAdapter): GraphQLInputType | string {
+    // TODO: Keep working here
```

Comment:
is there something else to be added here?

Reply:
No, I left that comment by mistake